production/loki-mixin/mixin-ssd.libsonnet

```diff
@@ -1,6 +1,4 @@
-(import 'dashboards.libsonnet') +
-(import 'alerts.libsonnet') +
-(import 'recording_rules.libsonnet') + {
+(import 'mixin.libsonnet') + {
```

Comment:
was this change automated?

Reply:
No i just figured there was no need to define it twice.

Reply:
we should just double check that everything still works here, I'll have to double check if we have an ssd deployment ourselves to deploy this against but if you have one yourself just lmk

Reply:
I have several SSD deployments running myself, and this update does not render anything that previously worked with SSD unusable now :)

I also noticed that I forgot to remove the import of config.libsonnet from dashboards.libsonnet, this is done now.

Previously, the only variables found in config.libsonnet were used in the dashboards. Therefore, it was properly decided to simply import it within the dashboards.libsonnet file. This change ensures its availability for alerts and recording rules as well.

Considering the only difference between mixin.libsonnet and mixin-ssd.libsonnet is some alterations to the _config+:: object and grafanaDashboardFolder, I believe it's acceptable to import everything from mixin.libsonnet and perform the necessary overwrites.

Alternatively, we could completely remove mixin-ssd.libsonnet and allow users to specify their preferences in config.libsonnet.